### PR TITLE
first commit for adjust_cohort data, add cohort job to DAG, update de…

### PR DESCRIPTION
**Pull request by Marlene-M-Hirose**
…scription of deliverables metadata.yaml

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1255)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/adjust_derived/adjust_cohort_v1/query.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+        "device_types",
+    ]
+    # getting overview metrics for different kpis / Deliverables
+    url = f"https://api.adjust.com/kpis/v1/{app_token}/cohorts"  # overview
+    url_params = f"start_date={start_date}&end_date={end_date}&kpis={','.join(kpis)}&grouping={','.join(groupings)}"
+    headers = {
+        "Authorization": f"Bearer {api_token}",
+    }
+
+    response = requests.get(url, headers=headers, params=url_params)
```
Comment (gleonard-m) [inline sql/moz-fx-data-shared-prod/adjust_derived/adjust_cohort_v1/query.py:96]: Check the value of response.status_code != 200 and log an error.  Specifically check for 401 (Auth)  and Bad Request (400)

File: sql/moz-fx-data-shared-prod/adjust/adjust_kpi_deliverables/view.sql [hunk 2]
```diff
@@ -2,7 +2,7 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.adjust.adjust_kpi_deliverables`
 AS
 SELECT
-    date as submission_date,
+    date,
```
Comment (gleonard-m) [inline sql/moz-fx-data-shared-prod/adjust/adjust_kpi_deliverables/view.sql:5]: why was this changed to a more generic column name?

Comment (Marlene-M-Hirose) [reply sql/moz-fx-data-shared-prod/adjust/adjust_kpi_deliverables/view.sql:5]: Mostly semantics. This date is when the data was aggregated rather than when it was submitted.

**Review by gleonard-m — APPROVED** [review]
LGTM
